### ISSUE TITLE
Black Ops 4 Stats

### ISSUE THREAD

**Issue by ohhsodead** [Tustin/cod-csharp#1]
Will you be updating the repo for the release of BO4 stats? I see that they're already accessible through their API. Otherwise I'll add support and open a pull request.

Thanks.

**Comment by Tustin** [comment]
Yeah I'll add that. I noticed they added the endpoint a week or two ago but I was a bit busy to implement it.

**Comment by ohhsodead** [comment]
Great, looking forward for it.
